sdk/python/kfp/components/component_factory_test.py

```diff
@@ -45,6 +47,15 @@ def test_with_packages_to_install_with_pip_index_url(self):
         for package in packages_to_install + pip_index_urls:
             self.assertTrue(package in concat_command)
 
+    def test_output_named_Output(self):
```

Comment:
This test looks good.

Let's move this to its own test class, as it doesn't fall within the group of tests that check the packages to install command ("TestGetPackagesToInstallCommand").